docs: second lol ppp (#48)
itchysats/10101: ppp/iteration_2_24102022.md

### Plans 📆

> - Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)

Reviewer: do you mean 'create full-fledged mock-ups' by "polishing them"?

Author: do we need to be specific? 🙈 I meant getting it to a point where we feel comfortable implementing it.

> - Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)

Reviewer 2: 🔧 I associate "reiterating" with saying something multiple times, although it's not wrong according to dictionaries. 

```suggestion
- Iterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```

> - Investigating ldk integration to the flutter mobile app. A little bit prototyping. 🚀

Reviewer 2: 🔧 Just some tweaks

```suggestion
- Investigate LDK integration with the Flutter mobile app. A little bit of prototyping 🚀
```

> - Write up a post about adding dlc to lightning channels. 💪

Reviewer 2: 🔧 

```suggestion
- Write a post about adding DLCs to Lightning channels 💪
```

> ### Progress ✅

Reviewer 2: ❓ Can we use hyperlinks rather than explicit URLs?

Author: I don't think so, but I can't test unfortunately as I can't find the write story button on bolt.fun anymore 🙈

- Decided on a mobile technology stack using flutter and rust. https://github.com/itchysats/10101/discussions/5
- Created initial mobile wallet project using flutter, rust and flutter_rust_bridge. https://github.com/itchysats/10101/issues/9
- Created wireframes of the user journey for opening and closing a non-perpetual CFD position. (TODO: add link to post here!)
- Wrote bolt.fun post about the first draft of the mobile user journey opening and closing non-perpetual CFD positions.

> - Decided on a design approach for integrating dlc and lightning channels. https://github.com/itchysats/10101/issues/8

Reviewer 2: 🔧 

```suggestion
- Decided on a design approach for integrating DLCs into Lightning channels. https://github.com/itchysats/10101/issues/8
```
